taccsite_cms/static/site_cms/css/src/site.css

```diff
@@ -4,16 +4,19 @@
 /* SEE: https://confluence.tacc.utexas.edu/x/IAA9Cw */
 
 /* SETTINGS */
-/* Shared with Portal */
 @import url("_imports/settings/border.css");
-/* Unique to CMS */
 @import url("_imports/settings/color.css");
-@import url("_imports/settings/props.max-width.css");
-@import url("_imports/settings/props.space.css");
 @import url("_imports/settings/font.css");
+@import url("_imports/settings/space.css");
+@import url("_imports/settings/max-width.css");
+/* Unique to CMS */
+@import url("./_imports/settings/color.css");
+@import url("./_imports/settings/font.css");
+@import url("./_imports/settings/space.css");
```

Comment:
(removed `props` filename prefix in TACC/Core-Styles#17 [`…max-width.css`](https://github.com/TACC/Core-Styles/pull/17/files#diff-0d662592406f132365a8223f05a1af57f24674325a1aed265e53136ed8819f70) and [`…space.css`](https://github.com/TACC/Core-Styles/pull/17/files#diff-05ef9222be1c3e7e67c68e1fce3c5dae9b3122e93bb8cba09bee8f7daadcb48a))